crates/nostr-mls/src/groups.rs

```diff
@@ -514,6 +514,112 @@ where
         }
         Ok(true)
     }
+
+    /// Batch add members
+    pub fn add_members(
+        &self, group_id: &GroupId,
+        key_packages: &[KeyPackage],
+    ) -> Result<(Vec<u8>, Vec<u8>), Error> {
```

Comment:
Instead of returning a tuple, I would return a struct (like for `self_update`). The same for the other new added methods